src/pages/configeditor/download_openshift.md

```diff
@@ -271,11 +271,11 @@ Custom properties include:
 
 - `com_ibm_watson_health_car_auth_logout_cookies` Set this property to the cookies your authentication method uses to store session information. If there are multiple cookies, comma-separate them. These will be cleared in the logout process.
 
-## Deploying to Custom ACD Configuration Editor Environments (optional)
+## Deploying Cartridges to ACD Installations (optional)
 
 ACD Configuration Editor provides the ability to deploy published cartridges to ACD instances in production environments.  To configure additional custom environments, the following files must be modified and the Cartridge microservice must be redeployed using Helm (see Installing ACD Configuration Editor instructions above).
 
-1. `merative-acd-ce/crtg/chart/crtg/rev-proxy.conf`
+1. `merative-acd-ce/crtg/chart/crtg/nginx-server.conf`
```

Comment:
Line 319 and 337 also need updating with a similar change from rev-proxy.conf to nginx-server.conf.

Formatting on line 337 and 338 doesn't appear correct. It is providing 2 different files, but hard to tell that with the single backtick around each since the gray box behind them seems to run together.